docs/library/machine.Pin.rst

```diff
@@ -191,7 +191,8 @@ Methods
    The arguments are:
 
      - ``handler`` is an optional function to be called when the interrupt
-       triggers.
+       triggers. The handler must take exactly one argument. Its meaning is
+       platform specific.
```

Comment:
I'm pretty sure the argument to the handler is the pin object itself, the one that triggered the IRQ.  So might be better to specify this fact.

Reply:
Now done.